handlers/captcha.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * This file is part of Moderent.
+ *
+ * Moderent is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Moderent is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Afferto General Public License
+ * along with Moderent.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Not related to this PR ig, but there is an extra space in this line :D

Reply:
you mean before the "If"?

Reply:
Yes

Reply:
It's GNU's. I'd rather leave it like that.